### DISCUSSION TITLE
Waveshare ESP32-S3 4.3" touch LCD display

### DISCUSSION THREAD

**Discussion by madiinfo** [arendst/Tasmota#21154] [Support request]
Hello insiders,
I have a Waveshare ESP32-S3 4.3" touch LCD display (https://www.waveshare.com/esp32-s3-touch-lcd-4.3.htm) and would like to run it with HASPmota. Can me someone please provide the required settings?
Thank you very much in advance. MFG MaDi

**Comment by Noschvie** [comment]
Have you got tasmota32-lvgl firmware installed and running?

**Reply by madiinfo** [reply]
Yes, but I don't get anything on the display. I've already tried all the available options. I think I need a display.ini file.

**Comment by lboue** [comment]
> Hello insiders, I have a Waveshare ESP32-S3 4.3" touch LCD display (https://www.waveshare.com/esp32-s3-touch-lcd-4.3.htm) and would like to run it with HASPmota. Can me someone please provide the required settings? Thank you very much in advance. MFG MaDi

I'd like to do the same with my board

**Comment by wjohn007** [comment]
I use this almost identical panel
https://www.waveshare.com/product/esp32-s3-touch-lcd-4.3b.htm

I am not able to get it to work under Tasmota. (Display)

With espHome it worked. The exotic IOExpander CH422G is also supported here.
The opto-isolated IOs all work, as does the Modbus connection via the available RS485 terminals.

This was my first attempt with espHome and it makes me want more.
